lib/prompt.js

```diff
@@ -14,7 +14,11 @@ function promptMasked (options) {
     stdin.setEncoding('utf8')
     stderr.write(ansi.eraseLine)
     stderr.write(ansi.cursorLeft)
-    stderr.write(options.prompt)
+    if (cli.console.mocking()) {
```

Comment:
sorry about the back and forth here, but you can just do `cli.console.writeError(options.prompt)`